Doc/whatsnew/3.13.rst

```diff
@@ -170,6 +170,9 @@ pathlib
   :exc:`NotImplementedError` when a path operation isn't supported.
   (Contributed by Barney Gale in :gh:`89812`.)
 
+* Add :meth:`pathlib.Path.from_uri` classmethod.
```

Comment:
I think this needs an article:
```suggestion
* Add the :meth:`pathlib.Path.from_uri` classmethod.
```

Reply:
Hm, that doesn't appear in-keeping with the way other new functions/methods are mentioned in the document. Would this work?

```suggestion
* Add :meth:`pathlib.Path.from_uri` classmethod that creates a path object from a 'file' URI.
```

Reply:
> Hm, that doesn't appear in-keeping with the way other new functions/methods are mentioned in the document

*What's New* will be edited quite a lot before October next year -- I think it's better to do both for better grammar and style (just think of your GCSE English teacher!)

Perhaps:

```suggestion
* Add :meth:`pathlib.Path.from_uri`, a new constructor to create a :class:`pathlib.Path`
  object from a 'file' URI (``file:/``).
```